### DISCUSSION TITLE
i need Github learning resources .

### DISCUSSION THREAD

**Discussion by SMGolamAlif** [community/community#78323] [New to GitHub] [answered]
Hey there! 🌟 I'm super pumped to start my coding adventure and dive into the fascinating world of **GitHub** and **Git**. I've got this burning curiosity to figure out how these things actually work and what all the cool kids are talking about.

## My Coding Wishlist:
- Get the lowdown on what makes GitHub tick.
- Wrap my head around the mysteries of Git.
- Explore the magical GitHub products (seriously, what's the hype?).
- Nailing down how to use all these tools (it's a bit overwhelming, gotta admit).
- Trying not to get lost in the world of mysterious Git commands.

But uh-oh, here's the snag: I'm having a bit of a treasure hunt trying to find beginner-friendly resources. 🕵️‍♂️ Can you lend a hand and point me in the right direction? I'm all ears and super ready to level up my coding game! 🚀✨

**Comment by leeringrose** [comment, marked as answer]
Good decision!
I would recommend you follow all the steps on https://skills.github.com.

Also, I thought this would be helpful for the start of the Github journey to understand Github's awesome products more.
https://github.com/sachin-source/top-github-repositories-which-everyone-should-look

**Reply by SMGolamAlif** [reply]
Thank you so much .
